src/shared/interfaces/json-schema.ts

```diff
@@ -0,0 +1,36 @@
+import { SchemaOption } from './schema-option';
+
+export interface JSONSchema extends SchemaOption {
```

Comment:
I got it somewhere like this link but, simplified it. My idea is to add the keys as we need/support them to keep it simple.